Automattic/newspack-newsletters
# fix(social-icons): add x.com icon; filter social icons block icons

### All Submissions:






### Changes proposed in this Pull Request:



1. Adds social icon support for x.com
2. Adds filtering of what's available in the `core/social-links` block, so it matches what can be rendered by the plugin

### How to test the changes in this Pull Request:

1. Insert a Social Icons blocks in a newsletter
3. Observe only services from [this list](https://github.com/Automattic/newspack-newsletters/blob/80f3447adb62daa45338cd78cee2ac38d1c9d4cd/includes/class-newspack-newsletters-renderer.php#L157-L165) can be added (e.g. Amazon won't be there)
4. Insert x.com and some other links 
5. Send a test newsletter - observe the icons are rendered as expected

### Other information:




---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1207372376329593
